extensions/ql-vscode/src/model-editor/model-editor-view.ts

```diff
@@ -731,7 +748,25 @@ export class ModelEditorView extends AbstractWebview<
               asError(e),
             )`Failed to auto-run generating models: ${getErrorMessage(e)}`,
           );
+          return;
         }
+
+        progress({
+          step: 4000,
+          maxStep: 4000,
+          message: "Saving generated models",
+        });
+
+        const fileContents = `# This file was automatically generated based from ${this.databaseItem.name}. Manual changes will not persist.\n\n${modelExtensionFileToYaml(extensionFile)}`;
```

Comment:
Minor suggestion:

```suggestion
        const fileContents = `# This file was automatically generated from ${this.databaseItem.name}. Manual changes will not persist.\n\n${modelExtensionFileToYaml(extensionFile)}`;
```

Or maybe change `from` to `on` or something like that instead?